source/style-guide/style/titles-and-headings.txt

```diff
@@ -285,5 +285,22 @@ contents (TOC) for a collection of content:
    heading for the TOC. An alternate TOC title or heading should convey
    the same intent of the full title or heading.
 
+   .. example::
+
+      .. code-block:: rst
+         :linenos:
+         :emphasize-lines: 4-6, 9-10
+
+         .. toctree::
+            :titlesonly:
+
+            Install the Operator </installation>
+            Deploy Ops Manager Resources </om-resources>
+            Deploy MongoDB Database Resources </mdb-resources>
+            /tutorial/modify-resource-image
+            /reference
+            Release Notes </release-notes>
+            MongoDB Community Kubernetes Operator <https://github.com/mongodb/mongodb-kubernetes-operator>
+
```

Comment:
Ah, sorry, I didn't mean as a TOC entry, though this example is indeed useful.

I meant only the `:ref:` directive, where renaming away from the default heading link is common practice in Serverland. Ping me if you want to talk more, of feel free to disregard if `:ref:` doesn't need a callout here.

Reply:
Not here. I'll cover it in the markup guide.